php/web/src/AppBundle/Entity/Character.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace AppBundle\Entity;
+
+use NoGame\Assertion;
+use Ramsey\Uuid\Uuid;
+
+final class Character
+{
+    /**
+     * @var Uuid
+     */
+    private $id;
+
+    /**
+     * @var Uuid
+     */
+    private $accountId;
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @var string
+     */
+    private $normalizedName;
+
+    /**
+     * @var int
+     */
+    private $health;
+
+    /**
+     * @var int
+     */
+    private $experience;
+
+    /**
+     * @param Uuid $accountId
+     * @param string $name
+     * @param int $health
+     */
+    public function __construct(Uuid $accountId, string $name, int $health)
+    {
+        $this->id = Uuid::uuid4();
```

Comment:
as you can see Player.js is still generated on the fly by Protocol.js - when I move to reading Players from storage I will remove generating id from Player.js